pkgs/tools/misc/urjtag/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pythonBindings ? false
+, jedecSupport ? false
+, python3 ? null
+, stdenv, autoconf, automake, pkgconfig, gettext, intltool, libtool, bison, flex, fetchgit, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "urjtag";
```

Comment:
There should be a version number here; "urjtag-VERSION". If upstream doesn't make releases, use the date (YYYY-MM-DD) of the git commit.